src/Objs/Redirect/RedirectComponent.tsx

```diff
@@ -14,13 +14,13 @@ provideComponent(Redirect, ({ page }) => {
   const link = page.get('link')
 
   React.useEffect(() => {
-    if (!link) return
-
     if (requireLogin && !isUserLoggedIn()) {
       ensureUserIsLoggedIn()
       return
     }
 
+    if (!link) return
+
```

Comment:
* [ ] FYI This change will trigger `ensureUserIsLoggedIn()` twice. I liked the code before more, because it only triggered it once after the link target was loaded. Why is this important, maybe add a comment?:
> Bugfix: RedirectComponent ensure log in before link check

Reply:
Because a not logged in user with have `link === null` since the user can't read the protected content.